helix-common/src/main/java/org/apache/helix/HelixConfigStringProcessUtil.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.helix;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class HelixConfigStringProcessUtil {
```

Comment:
There are commonly-followed Java idiom for creating util classes (`final` class, private constructor, etc.). Could we follow that? https://www.delftstack.com/howto/java/java-utility-class.java/

Reply:
This is a good suggestion. Let's follow this.

Reply:
TFTR. Updated